### PR TITLE
Farm 35 creation page cockpit

### PR DESCRIPTION
**Pull request by AntoineCAU**

### CODE REVIEW

**Review by Erriep — DISMISSED** [review]

File: src/components/Cockpit/index.jsx [hunk 1]
```diff
@@ -1,10 +1,72 @@
...
+            N’hésitez pas à nous contacter pour nous dire ce que vous pensez de
+            l’application ou si vous avez des idées d&apos;amélioration.
+          </p>
+          <button
+            className="contactButton"
+            onClick={handleClickContact}
+            type="button"
+          >
+            Donner mon avis
+          </button>
```
Comment (LoicBrassart) [inline src/components/Cockpit/index.jsx]: Why not use a <Link> ?

File: src/components/Cockpit/index.jsx [hunk 2]
```diff
@@ -1,10 +1,72 @@
...
+              <p className="completed">Votre profil est complété !</p>
+            )}
+            {!completed && (
+              <p className="notCompleted">Votre profil est incomplet !</p>
+            )}
+            <button
+              className="contactButtonProfil"
+              onClick={handleClickProfile}
+              type="button"
+            >
```
Comment (LoicBrassart) [inline src/components/Cockpit/index.jsx]: Why not use a <Link> ?

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Use Links instead of buttons

**Review by Erriep — APPROVED** [review]

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**